src/common/telemetryHelper.ts

```diff
@@ -52,6 +52,14 @@ export class TelemetryHelper {
         Telemetry.send(event);
     }
 
+    public static sendDirect(eventName: string, properties?: Telemetry.ITelemetryProperties): void {
+        const event = TelemetryHelper.createTelemetryEvent(eventName, {isDirect: true});

```

Comment:
'isHermes' would be a better name to understand more clearly what the telemetry is for.